### PR TITLE
Replace Chaos Event System with ICES - Intensity Credit Event System

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Removes the modular Chaos event/voting system. Replaces with a modified version of the TG weighted system called Intensity Credit.

Puts all SR event weights in a singular config file. LOW/MEDIUM/HIGH chaos on the modular system is broken down into 5 weighting levels.

Intensity Credits are accumulated as the round progresses, and spent by the events system on 'intense' events.

https://hackmd.io/@lessthanthree/HJKVFH0yn

## How This Contributes To The Skyrat Roleplay Experience
Using the event voting system was a failure disabled a long time ago. Since we're not using that functionality, we don't need a modular system doing that. Time to semi-reinvent the wheel. Some events that were not firing under Chaos have been re-added, radiation leak disabled.

## Changelog

:cl: LT3
del: Removed Chaos Event System
add: New! Intensity Credit Event System
fix: Admins can actually cancel/reroll all random events
balance: Frequency of all random events have been adjusted. To what? It is a mystery...
/:cl:

### CODE REVIEW

**Review by Paxilmaniac — COMMENTED** [review]
modular_skyrat\modules\event_vote\code\event_chaos_system.dm is not included

Did you remember to include all your files?

Otherwise:

If this file in particular isn't going to be used anymore, delete it.

**Comment by lessthnthree** [comment]
> s not included
> 
> Did you remember to include all your files?

It was meant to be deleted, fixed.

File: modular_skyrat/modules/events/code/event_frequency.dm [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+	frequency_upper = 25 MINUTES
+
+/datum/round_event_control/abductor
+	weight = VERY_LOW_EVENT_FREQ
+
+// Disabled: Part of dynamic.
+/datum/round_event_control/alien_infestation
+	max_occurrences = 0
+
+/datum/round_event_control/anomaly/anomaly_flux
```
Comment (Zonespace27) [inline modular_skyrat/modules/events/code/event_frequency.dm]: aren't a lot of these set by default?

File: modular_skyrat/modules/cme/code/cme.dm [hunk 2]
```diff
@@ -121,6 +121,8 @@
 	else
 		switch(cme_intensity)
 			if(CME_UNKNOWN)
+				if(SSsecurity_level.get_current_level_as_number() < SEC_LEVEL_RED)
```
Comment (Zonespace27) [inline modular_skyrat/modules/cme/code/cme.dm]: atomize these changes out, they have nothing to do with the PR

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/cme/code/cme.dm (hunk 2 above)
Comment (lessthnthree) [reply modular_skyrat/modules/cme/code/cme.dm]: Okay

File: modular_skyrat/modules/events/code/event_frequency.dm (hunk 1 above)
Comment (lessthnthree) [reply modular_skyrat/modules/events/code/event_frequency.dm]: Yes, but for organization purposes this is easier to manage/edit as events get added and removed over time. Particularly added.

**Review by Gandalf2k15 — APPROVED** [review]
I'm merging this.

**Pull request closed without merging**